sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_custom_model.py

```diff
@@ -18,6 +18,9 @@
 
 class TestDACAnalyzeCustomModel(FormRecognizerTest):
 
+    def teardown(self):
+        self.sleep(4)
```

Comment:
4 seconds keeps the pipeline run around 2 hours, while reducing the frequency of errors.

Reply:
Live pipeline run: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1410278&view=results